project/Maven.scala

```diff
@@ -38,9 +36,9 @@ object Maven {
 
   /** Runs maven commands (blocking) on given project directory (by cd-ing into it) */
   def mvnRun(javaHome: Option[File], proj: String, commands: String*): Unit = {
-    val exportJHome = javaHome.map(h ⇒ s"""export JAVA_HOME="$h" """).getOrElse("")
-    val initialCmd = s"$exportJHome; cd $proj; mvn "
-    
+    val exportJHome = javaHome.map(h ⇒ s"""export JAVA_HOME="$h"; """).getOrElse("")
```

Comment:
Sadly, it will not. We can probably use sbt process library to have a platform independent solution. However this PR only moves a semicolon around. We can address cross-platformability in another PR.